ui/next/config.js

```diff
@@ -137,6 +139,9 @@ System.config({
     "github:jspm/nodelibs-zlib@0.1.0": {
       "browserify-zlib": "npm:browserify-zlib@0.1.4"
     },
+    "npm:ByteBuffer@0.2.6": {
```

Comment:
Ok I uninstalled `ByteBuffer` and `bytebuffer`, then reinstalled `bytebuffer` directly (not through `npm`) which all seems to work.